### PR TITLE
Feature/add delete button to attachments

### PR DESCRIPTION
**Pull request by kevmodrome**
## Description
- Adds a delete file action on the attachment component in the grid
- Adds modal confirm screens to Delete Rows and delete File actions on the grid component

## Screenshots
https://cln.sh/pzHsRD

### CODE REVIEW

File: packages/standard-components/src/DataGrid/AttachmentCell/Button.svelte [hunk 1]
```diff
@@ -1,18 +1,18 @@
 <script>
-  import { createEventDispatcher } from "svelte"
-  import { DropdownMenu, TextButton as Button, Icon } from "@budibase/bbui"
+  // import { createEventDispatcher } from "svelte"
```
Comment (shogunpurple) [inline packages/standard-components/src/DataGrid/AttachmentCell/Button.svelte:2]: Let's delete the commented out code if it's not required.

**Review by shogunpurple — APPROVED** [review]
LGTM 🚀 Just on the design - I think the X needs to be aligned vertically and horizontally and possibly made a little smaller - the alignment looks a little off.

**Review by aptkingston — APPROVED** [review]
LGTM! 👌

**Comment by aptkingston** [comment]
Weird error that failed it when running in CI though... Looks like a random failure just.
`[3727:1015/135147.422346:FATAL:gpu_data_manager_impl_private.cc(439)] GPU process isn't usable. Goodbye.`

**Pull request merged**